ci-operator/jobs/openshift/kubernetes/openshift-kubernetes-master-presubmits.yaml

```diff
@@ -842,70 +972,50 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-gcp,?($|\s.*)
   - agent: kubernetes
-    always_run: false
+    always_run: true
     branches:
     - master
-    cluster: build01
```

Comment:
This breaks rehearsals

Reply:
yep, I was missing `sanitize-prow-jobs` invocation